manual_tests/test_rec_int_for_BB_data_cts.Rmd

```diff
@@ -24,8 +24,10 @@ library(LAGO)
 ## Load the BetterBirth data with EBP proportions
 ```{r}
 # BB data has been open sourced, so I pre-loaded the BB data in the LAGO package 
-BB_proportions <- LAGO::BB_proportions
-head(BB_proportions)
+#BB_proportions <- LAGO::BB_proportions
+#head(BB_proportions)
+load("/Users/minhthubui/Desktop/LAGO-R-Package/data/BB_proportions.rda")
+BB_proportions <- get("BB_proportions")
```

Comment:
yeah, try devtools:load_all(). you may need to restart R and open the folder as a project first.